runelite-client/src/main/java/net/runelite/client/plugins/menuentryswapper/MenuEntrySwapperPlugin.java

```diff
@@ -569,6 +573,10 @@ else if (config.swapTeleportItem() && option.equals("wear"))
 			swap("rub", option, target, true);
 			swap("teleport", option, target, true);
 		}
+		else if (config.swapMtaItems() && MTA_ITEMS.contains(eventId) && Arrays.asList("wield", "wear").contains(option))
```

Comment:
Don't construct an array each time the full conditional runs. Either define the array as a static variable, or use two separate comparisons. (ie. `("wield".equals(option) || "wear".equals(option))`)